app/code/Magento/InventoryApi/Api/Data/SourceInterface.php

```diff
@@ -0,0 +1,320 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\InventoryApi\Api\Data;
+
+use \Magento\Framework\Api\ExtensibleDataInterface;
+
+/**
+ * @api
+ */
```

Comment:
please provide a description for Interface
Each interface must have a precise and clear description